WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -8,6 +8,8 @@ import Yosemite
 final class AuthenticationManagerTests: XCTestCase {
     private var navigationController: UINavigationController!
     private let window = UIWindow(frame: UIScreen.main.bounds)
+    private var analyticsProvider: MockAnalyticsProvider!
+    private var analytics: Analytics!
```

Comment:
Are these needed? 😅

Reply:
Nope, my mistake here. It was supposed to be introduced in the unit tests PR 😅 Thanks!